src/lib/components/SgtBreadCrumb/SgtBreadCrumb.vue

```diff
@@ -82,6 +81,10 @@ export default class SgtBreadCrumb extends Vue {
   public routeChangeHandler() {
     this.populatePathList();
   }
+
+  capitalizeFirstLetter = (value: string) => {
```

Comment:
Why is it declared separately here instead of using the one that is in the common utils?

Reply:
I have removed service folder from lib so to make this as a component library only, this functions is kept here so now it is not dependent on any other service or file. (In case we have more such functions we could create ts file inside the SgtBreadCrum component.)

Reply:
What if I need to use this 'capitalizeFirstLetter' function in some other Sgt component?

Reply:
Still need to think of that. how libraries should be scaffolded and how the common function should be designed. as of now most of the components are independent.